galaxy.yml

```diff
@@ -8,7 +8,7 @@ description: Ansible collection for deploying Kubernetes clusters
 license:
   - GPL-3.0-or-later
 dependencies:
-  ansible.posix: ">=1.3.0"
+  ansible.posix: "==1.3.0"
```

Comment:
In the past, we used to lock things but that would cause issues for our downstream users due to the fact that it makes it hard to consume this library.

I think we should try and fix the `ansible.posix` issue to be honest.